undead-discord.css

```diff
@@ -255,7 +248,7 @@
   width: 500px !important;
 }
 
-.chat > .title-wrap > .title .channel-name {
+.chat > .title .channel-name {
   line-height: 25px !important;
 }
```

Comment:
This change was made because letters like `g` were cut off in the channel name. If you can confirm that these letters are not cut off anymore, please just remove this whole block.

Reply:
They don't seem to cut off any channel names with that letter. I'll make a push and let you try it out